docs/developer/DEVELOP.md

```diff
@@ -786,3 +789,13 @@ Sometimes, the secrets found by the tool can be false-positives, if you believe
 Django signals are globally disabled via an autouse pytest fixture in the
 global `conftest.py` file, in order to enable them for a specific test
 you can mark your test with the `@pytest.mark.enable_signals` decorator.
+
+### Configuring CORS Rules for API Access
+
+In order to permit API access from different domains, Cross-Origin Resource Sharing (CORS) is configured using the corsheaders Django app. This is crucial for development scenarios where changes to CORS rules are necessary, and for operations to load in CORS URLs correctly.
+
+Ensure that the OSIDB_CORS_ALLOWED_ORIGINS environment variable is set correctly when deploying the application, for instance:
+
+The OSIDB_CORS_ALLOWED_ORIGINS environment variable should be set in the format `["http://example-ui1.com", "http://example-ui2.com"]`.
```

Comment:
There should probably be an indentation.

Reply:
Not sure what you meant by this but I made some changes to keep it <80col

Reply:
I would prefer:
```
Ensure that the OSIDB_CORS_ALLOWED_ORIGINS ..., for instance:
- do this
```
instead of :
```
Ensure that the OSIDB_CORS_ALLOWED_ORIGINS ..., for instance:
do this
```

Reply:
Long lines are ok, but I would expect a list of possibilities when seeing `..., for instance:`. 

But you can ignore this whole thread and keep the original version.

Reply:
Since it's not a list of possibilities I'll leave it as it is